### PR TITLE
update SplicedBinnedPareto demos from nursery version to gluonts version

### PR DESCRIPTION
**Pull request by elenaehrlich**
*Issue #, if available:* 1410

*Description of changes:* Updated the SBP example notebooks from the nursery version to the gluonts version. Also, included a small fix to ensure the tail_percentile_gen_pareto to be on same device as logits. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by elenaehrlich** [comment]
Demo notebook for gluonts-compatible SplicedBinnedPareto now added

**Review by lostella — CHANGES_REQUESTED** [review]
@elenaehrlich looks good to me! Only one thing: maybe it is better to commit the notebooks without the cells outputs (so clearing all outputs before committing), would that be ok?

**Comment by elenaehrlich** [comment]
> 

Thank you Lorenzo, I've cleared the notebooks' outputs and re-pushed

**Review by lostella — APPROVED** [review]
thanks @elenaehrlich! 🚀
